modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -396,17 +466,21 @@ public CodegenParameter fromRequestBody(RequestBody body, Set<String> imports, S
         CodegenProperty modelProp = fromProperty("body", realSchema);
         if (modelProp.isPrimitiveType && (modelProp.hasValidation || modelProp.isEnum)) {
             String simpleDataType = result.dataType;
-            result.isPrimitiveType = false;
```

Comment:
This is intentional.
We are deleting this because in api_doc.mustache we using isModel to determine if this is a model or not. We generate models for models of type object and primitive types like string/number etc which have validations or enums. So the mustache code is not depending upon isPrimitiveType having any specific value here, so we don't need to set it anymore.

Reply:
The postProcessModelProperty is handling model properties  and this code is handling response bodies. 
- In model_doc.mustache we are switching on isPrimitiveType to determine if a property is a model or not
- in api_doc we are switching on isModel to determine if a property is a model or not
@jimschubert do you want me have both of these mustache files switch on isPrimitiveType?

Reply:
@jimschubert I changed the templates to only link to model docs for
- model docs if complexType is not null
- api docs if baseType is not null
This way we don't have to worry about isModel/isAlias etc.